src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -68,6 +85,12 @@ end subroutine read_geogrid
    integer (kind=I1KIND), dimension(:), pointer :: hlanduse ! Dominant land mask (0 or 1)
    real (kind=RKIND) :: hc   ! critical height
 
+   ! For each point in the box, contains x and y coordinates of the global pixel
+   integer, dimension(:), pointer :: local_box_x  
+   integer, dimension(:), pointer :: local_box_y  
+   integer :: max_local_tiles                    ! Max possible tiles required by current rank
+   integer :: cum_local_tiles                    ! Total number of tiles required by current rank
```

Comment:
It looks like `max_local_tiles` and `cum_local_tiles` may have been used for some early investigation, but I think these can be removed now.

Reply:
Yes, it's not really necessary with this approach. Removing